src/wdio/conf.js

```diff
@@ -10,6 +10,12 @@ exports.config = {
   capabilities: [
     {
       browserName: 'chrome',
+      axe: {
```

Comment:
I actually meant the inverse, why are you putting the config in the capabilities?

Reply:
as you see in your comment below, I have the config in the capabilities as they are passed into the before function instead of the whole config so I have access to them.  Switching to use onPrepare or another function that allows the config to be passed in will not give us access to the global browser object that we are applying the axe config too.

Reply:
you aren't applying it to the global browser object. Your saving off the config as an instnace variable in the AxeService. Then when the browser command injects the axe config, it can call `axe.configure` with that config.